lib/main.js

```diff
@@ -60,6 +60,7 @@ export default {
     const Composer = require('./composer')
 
     global.latex = new Latex()
+    latex.opener.initializeOpeners()
```

Comment:
Ok. I'll have to dig into the package activation issue to make it happen. `OpenerRegistry` is getting created in a whole bunch of places in the specs. Moving it back won't cause spec failure, just console messages, BTW.